concepticondata/references/references.bib

```diff
@@ -3102,11 +3102,14 @@ @Article{Yao2017
   Number                              = {4}
 }
 
-@book{Hartmann2013,
-    publisher = {Max Planck Institute for Evolutionary Anthropology},
-    title = {Valency Patterns Leipzig},
-    url = {http://valpal.info},
-    address = {Leipzig},
-    editor = {Hartmann, Iren and Haspelmath, Martin and Taylor, Bradley},
-    year = {2013}
+@Book{Grierson1928,
```

Comment:
@phylostar, to get your PR in line with the current state in concepticon, you only need to add the reference of Hartman2013, as you see it in the diff here to your references.bib file, and everything is fine, can you please do that? When done, we can finally merge this PR.